build.php

```diff
@@ -0,0 +1,125 @@
+<?php
```

Comment:
For future consideration, as this changes/grows, I've had success with the [symfony/console component](https://symfony.com/doc/current/components/console.html) which makes interacting with the command line input easier.

Reply:
Good note! I like it! We'll do that in another PR.